src/components/Board/Board.js

```diff
@@ -1,27 +1,45 @@
 import React, { useEffect, useState } from 'react';
 import Grid from '@material-ui/core/Grid';
 import Column from '../Column/Column';
-import * as cardsApi from '../../api/cardsApi';
+import { getBoards } from '../../api/boardsApi';
+import { addNewCard } from '../../services/BoardService';
 
 const Board = () => {
-  const [cards, setCards] = useState([]);
-
+  const [board, setBoard] = useState({ columns: [] });
+  const [error, setError] = useState(false);
   const addCard = (newCard) => {
-    setCards([...cards, newCard]);
+    const newState = addNewCard(board, newCard);
+
+    setBoard({ ...board, newState });
   };
 
   useEffect(() => {
-    cardsApi.getCards().then((cardsResponse) => {
```

Comment:
Please remove the `getCards` function from the `cardsApi.js` file since it's no longer used in the project.

Reply:
done